src/package.lisp

```diff
@@ -142,6 +142,7 @@
   ;; apply-gate.lisp
```

Comment:
we need to export the various kinds of new states under a new heading, along with anything useful (like functions and methods on them)